base/requirements.txt

```diff
@@ -1,29 +1,29 @@
-anonlink==0.13.1
-anonlink-client==0.1.4
+alembic==1.5.4
+git+https://github.com/data61/anonlink@2a886b36465d8b7e250563a42bc834a5f7046750#egg=anonlink
```

Comment:
I just started the anonlink 0.14.2 release process. It would look cleaner if we can reference that instead.